### PR TITLE
fix: Minor a11y fixes in v9 stories

### PR DESCRIPTION
**Pull request by smhigley**
Example-only fixes, primarily fixing accessible name issues in buttons and images. One addition of a `lang` attribute.

Fixes several existing ADO bugs.

### CODE REVIEW

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29785/perf-test-react-components/Avatar-mount_base.html">652</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29785/perf-test-react-components/Avatar-mount.html">660</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29785/perf-test-react-components/Button-mount_base.html">312</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29785/perf-test-react-components/Button-mount.html">323</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29785/perf-test-react-components/Field-mount_base.html">1147</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29785/perf-test-react-components/Field-mount.html">1166</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29785/perf-test-react-components/FluentProvider-mount_base.html">734</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29785/perf-test-react-components/FluentProvider-mount.html">719</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29785/perf-test-react-components/FluentProviderWithTheme-mount_base.html">85</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29785/perf-test-react-components/FluentProviderWithTheme-mount.html">89</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29785/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">70</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29785/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">62</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29785/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">83</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29785/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">78</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29785/perf-test-react-components/MakeStyles-mount_base.html">896</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29785/perf-test-react-components/MakeStyles-mount.html">859</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29785/perf-test-react-components/Persona-mount_base.html">1731</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29785/perf-test-react-components/Persona-mount.html">1722</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29785/perf-test-react-components/SpinButton-mount_base.html">1408</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29785/perf-test-react-components/SpinButton-mount.html">1387</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-alert</samp> <br /> <abbr title='bundle-size/Alert.fixture.js'>Alert</abbr>                                                                                       |  `84.293 kB`<br />`23.456 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/Button.fixture.js'>Button</abbr>                                                                                    |  `40.074 kB`<br />`11.292 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/CompoundButton.fixture.js'>CompoundButton</abbr>                                                                    |   `47.43 kB`<br />`12.787 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/MenuButton.fixture.js'>MenuButton</abbr>                                                                            |  `44.848 kB`<br />`12.668 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/SplitButton.fixture.js'>SplitButton</abbr>                                                                          |  `52.862 kB`<br />`14.254 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/ToggleButton.fixture.js'>ToggleButton</abbr>                                                                        |  `57.122 kB`<br />`13.194 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.All.fixture.js'>Card - All</abbr>                                                                                |  `92.955 kB`<br />`26.977 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.fixture.js'>Card</abbr>                                                                                          |  `87.781 kB`<br />`25.462 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardFooter.fixture.js'>CardFooter</abbr>                                                                              |   `13.423 kB`<br />`5.481 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardHeader.fixture.js'>CardHeader</abbr>                                                                              |   `15.666 kB`<br />`6.235 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardPreview.fixture.js'>CardPreview</abbr>                                                                            |   `14.374 kB`<br />`5.854 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr>              |   `70.013 kB`<br />`20.17 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `207.755 kB`<br />`59.417 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `42.297 kB`<br />`14.008 kB` |
| <samp>react-image</samp> <br /> <abbr title='bundle-size/Image.fixture.js'>Image</abbr>                                                                                       |   `16.096 kB`<br />`6.317 kB` |
| <samp>react-message-bar</samp> <br /> <abbr title='bundle-size/Default.fixture.js'>MessageBar (all components)</abbr>                                                         |    `25.54 kB`<br />`9.102 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr>                                                         |    `6.651 kB`<br />`2.252 kB` |
| <samp>react-provider</samp> <br /> <abbr title='bundle-size/FluentProvider.fixture.js'>FluentProvider</abbr>                                                                  |   `22.837 kB`<br />`8.434 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/d369aa60a3f778a18768c2fc5e94649066459c5d'>d369aa60a3f778a18768c2fc5e94649066459c5d</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

**Review by khmakoto — APPROVED** [review]

**Review by layershifter — APPROVED** [review]

File: packages/react-components/react-image/stories/Image/ImageFit.stories.tsx [hunk 1]
```diff
@@ -5,46 +5,86 @@ export const Fit = () => (
   <>
     <h1>None</h1>
     <div style={{ border: '1px solid green', height: 150, width: 300 }}>
-      <Image src="https://fabricweb.azureedge.net/fabric-website/placeholders/600x200.png" fit="none" />
+      <Image
+        src="https://fabricweb.azureedge.net/fabric-website/placeholders/600x200.png"
+        alt="Image placeholdder"
```
Comment (tudorpopams) [inline packages/react-components/react-image/stories/Image/ImageFit.stories.tsx:10]:
```suggestion
        alt="Image placeholder"
```

File: packages/react-components/react-image/stories/Image/ImageFit.stories.tsx [hunk 2]
```diff
@@ -5,46 +5,86 @@ export const Fit = () => (
...
+        fit="none"
+      />
     </div>
     <br />
     <div style={{ border: '1px solid green', height: 150, width: 300 }}>
-      <Image src="https://fabricweb.azureedge.net/fabric-website/placeholders/200x100.png" fit="none" />
+      <Image
+        src="https://fabricweb.azureedge.net/fabric-website/placeholders/200x100.png"
+        alt="Image placeholdder"
```
Comment (tudorpopams) [inline packages/react-components/react-image/stories/Image/ImageFit.stories.tsx:18]:
```suggestion
        alt="Image placeholder"
```

File: packages/react-components/react-image/stories/Image/ImageFit.stories.tsx [hunk 3]
```diff
@@ -5,46 +5,86 @@ export const Fit = () => (
...
+        fit="none"
+      />
     </div>
 
     <h1>Center</h1>
     <div style={{ border: '1px solid green', height: 150, width: 300 }}>
-      <Image src="https://fabricweb.azureedge.net/fabric-website/placeholders/600x200.png" fit="center" />
+      <Image
+        src="https://fabricweb.azureedge.net/fabric-website/placeholders/600x200.png"
+        alt="Image placeholdder"
```
Comment (tudorpopams) [inline packages/react-components/react-image/stories/Image/ImageFit.stories.tsx:27]:
```suggestion
        alt="Image placeholder"
```

File: packages/react-components/react-image/stories/Image/ImageFit.stories.tsx [hunk 4]
```diff
@@ -5,46 +5,86 @@ export const Fit = () => (
...
+        fit="center"
+      />
     </div>
     <br />
     <div style={{ border: '1px solid green', height: 150, width: 300 }}>
-      <Image src="https://fabricweb.azureedge.net/fabric-website/placeholders/200x100.png" fit="center" />
+      <Image
+        src="https://fabricweb.azureedge.net/fabric-website/placeholders/200x100.png"
+        alt="Image placeholdder"
```
Comment (tudorpopams) [inline packages/react-components/react-image/stories/Image/ImageFit.stories.tsx:35]:
```suggestion
        alt="Image placeholder"
```

File: packages/react-components/react-image/stories/Image/ImageFit.stories.tsx [hunk 5]
```diff
@@ -5,46 +5,86 @@ export const Fit = () => (
...
+        fit="center"
+      />
     </div>
 
     <h1>Contain</h1>
     <div style={{ border: '1px solid green', height: 200, width: 400 }}>
-      <Image src="https://fabricweb.azureedge.net/fabric-website/placeholders/400x200.png" fit="contain" />
+      <Image
+        src="https://fabricweb.azureedge.net/fabric-website/placeholders/400x200.png"
+        alt="Image placeholdder"
```
Comment (tudorpopams) [inline packages/react-components/react-image/stories/Image/ImageFit.stories.tsx:44]:
```suggestion
        alt="Image placeholder"
```

File: packages/react-components/react-image/stories/Image/ImageFit.stories.tsx [hunk 6]
```diff
@@ -5,46 +5,86 @@ export const Fit = () => (
...
+        fit="contain"
+      />
     </div>
     <br />
     <div style={{ border: '1px solid green', height: 250, width: 400 }}>
-      <Image src="https://fabricweb.azureedge.net/fabric-website/placeholders/400x200.png" fit="contain" />
+      <Image
+        src="https://fabricweb.azureedge.net/fabric-website/placeholders/400x200.png"
+        alt="Image placeholdder"
```
Comment (tudorpopams) [inline packages/react-components/react-image/stories/Image/ImageFit.stories.tsx:52]:
```suggestion
        alt="Image placeholder"
```

File: packages/react-components/react-image/stories/Image/ImageFit.stories.tsx [hunk 7]
```diff
@@ -5,46 +5,86 @@ export const Fit = () => (
...
+        fit="contain"
+      />
     </div>
     <br />
     <div style={{ border: '1px solid green', height: 200, width: 450 }}>
-      <Image src="https://fabricweb.azureedge.net/fabric-website/placeholders/400x200.png" fit="contain" />
+      <Image
+        src="https://fabricweb.azureedge.net/fabric-website/placeholders/400x200.png"
+        alt="Image placeholdder"
```
Comment (tudorpopams) [inline packages/react-components/react-image/stories/Image/ImageFit.stories.tsx:60]:
```suggestion
        alt="Image placeholder"
```

File: packages/react-components/react-image/stories/Image/ImageFit.stories.tsx [hunk 8]
```diff
@@ -5,46 +5,86 @@ export const Fit = () => (
...
+        fit="cover"
+      />
     </div>
     <br />
     <div style={{ border: '1px solid green', height: 200, width: 400 }}>
-      <Image src="https://fabricweb.azureedge.net/fabric-website/placeholders/400x300.png" fit="cover" />
+      <Image
+        src="https://fabricweb.azureedge.net/fabric-website/placeholders/400x300.png"
+        alt="Image placeholdder"
```
Comment (tudorpopams) [inline packages/react-components/react-image/stories/Image/ImageFit.stories.tsx:77]:
```suggestion
        alt="Image placeholder"
```

File: packages/react-components/react-image/stories/Image/ImageFit.stories.tsx [hunk 9]
```diff
@@ -5,46 +5,86 @@ export const Fit = () => (
...
+        fit="cover"
+      />
     </div>
     <br />
     <div style={{ border: '1px solid green', height: 200, width: 400 }}>
-      <Image src="https://fabricweb.azureedge.net/fabric-website/placeholders/600x200.png" fit="cover" />
+      <Image
+        src="https://fabricweb.azureedge.net/fabric-website/placeholders/600x200.png"
+        alt="Image placeholdder"
```
Comment (tudorpopams) [inline packages/react-components/react-image/stories/Image/ImageFit.stories.tsx:85]:
```suggestion
        alt="Image placeholder"
```

File: packages/react-components/react-image/stories/Image/ImageShadow.stories.tsx [hunk 10]
```diff
@@ -2,7 +2,11 @@ import * as React from 'react';
 import { Image } from '@fluentui/react-components';
 
 export const Shadow = () => (
-  <Image shadow src="https://fabricweb.azureedge.net/fabric-website/placeholders/300x300.png" />
+  <Image
+    shadow
+    src="https://fabricweb.azureedge.net/fabric-website/placeholders/300x300.png"
+    alt="Image placeholdder"
```
Comment (tudorpopams) [inline packages/react-components/react-image/stories/Image/ImageShadow.stories.tsx:8]:
```suggestion
        alt="Image placeholder"
```

File: packages/react-components/react-image/stories/Image/ImageFit.stories.tsx (hunk 1 above)
Comment (smhigley) [reply packages/react-components/react-image/stories/Image/ImageFit.stories.tsx:10]: oh good catch 😅

**Review by tudorpopams — APPROVED** [review]

**Pull request merged**